packages/manager/modules/pci/src/projects/project/storages/object-storage/users/users.controller.js

```diff
@@ -75,4 +75,20 @@ export default class PciStoragesContainersUsersController {
         ),
       );
   }
+
+  downloadRcloneFile(user) {
+    this.downloadOpenStackRclone(user);
```

Comment:
Suggestion: instead of calling `downloadOpenStackRclone()` here, we can directly call it on `on-click` in template.

Reply:
Okay Vikash added as per suggestion